app/component/itinerary/ItineraryDetails.js

```diff
@@ -42,6 +42,15 @@ import Emissions from './Emissions';
 import EmissionsInfo from './EmissionsInfo';
 import FareDisclaimer from './FareDisclaimer';
 
+import SecondaryButton from '../SecondaryButton';
+
+const followItineraryRoute = '../../../FollowTheItinerary';
+
+const followItinerary = e => {
+  e.stopPropagation();
+  window.open(followItineraryRoute, '_blank');
```

Comment:
Instead of window.open, you should usually use `context.router.push(path)`. However, I think in this case, you could use the `<Link>` component from the found library.